elide-core/src/main/java/com/yahoo/elide/utils/coerce/CoerceUtil.java

```diff
@@ -55,6 +54,11 @@ public static Object coerce(Object value, Class<?> cls) {
      */
     private static void setup() {
         BeanUtilsBean.setInstance(new BeanUtilsBean(new ConvertUtilsBean() {
+            {
+                // https://github.com/yahoo/elide/issues/260
+                // disable returning defaultValues when conversions fail
```

Comment:
This note should be changed from a negation to an assertion like: `enable throwing when conversion fails`